src/Services/PushService.ts

```diff
@@ -0,0 +1,80 @@
+import firebase from 'react-native-firebase';
+import {Alert} from 'react-native';
+import AsyncStorage from '@react-native-community/async-storage';
+
+let notificationListener;
+let notificationOpenedListener;
+let messageListener;
```

Comment:
More of a question than a comment; why are these listeners stored, will they be referenced later?